src/microbe_stage/Microbe.Interior.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Linq;
+using AngleSharp.Dom;
```

Comment:
I'm pretty sure this is a mistake, the microbe code should not be doing any HTML parsing...